dijet/config/config_run2.py

```diff
@@ -480,6 +480,65 @@ def make_jme_filename(jme_aux, sample_type, name, era=None):
             dataset.x.event_weights["normalized_muf_weight"] = get_shifts("muf")
             dataset.x.event_weights["normalized_pdf_weight"] = get_shifts("pdf")
 
+    # Trigger selection
+    # TODO: SingleJet triggers for AK8 and some special cases in UL16 & UL17
+    cfg.x.triggers = DotDict.wrap({
+        "dijet": {
+            "central": [
+                "DiPFJetAve40",
+                "DiPFJetAve60",
+                "DiPFJetAve80",
+                "DiPFJetAve140",
+                "DiPFJetAve200",
+                "DiPFJetAve260",
+                "DiPFJetAve320",
+                "DiPFJetAve400",
+                "DiPFJetAve500",
+            ],
+            "forward": [
+                "DiPFJetAve60_HFJEC",
+                "DiPFJetAve80_HFJEC",
+                "DiPFJetAve100_HFJEC",
+                "DiPFJetAve160_HFJEC",
+                "DiPFJetAve220_HFJEC",
+                "DiPFJetAve300_HFJEC",
+            ],
+        },
+        # TODO: single jet only for AK4 so far
+        #       Needed for UL17
+        "singlejet": {
+            "central": [
+                "PFJet40",
+                "PFJet60",
+                "PFJet80",
+                "PFJet140",
+                "PFJet200",
+                "PFJet260",
+                "PFJet320",
+                "PFJet400",
+                "PFJet450",
+                "PFJet500",
+                "PFJet550",
+            ],
+        },
+    })
+
+    cfg.x.trigger_thresholds = DotDict.wrap({
+        "dijet": {
+            "central": {
+                "UL17": [70, 87, 111, 180, 247, 310, 373, 457, 562],
+            },
+            "forward": {
+                "UL17": [73, 93, 113, 176, 239, 318],
+            },
+        },
+        "singlejet": {
+            "central": {
+                "UL17": [70, 87, 111, 180, 247, 310, 373, 457, 562],
+            },
+        },
+    })
+
```

Comment:
I notice you have a separate nesting level for the year in your threshold dict (at the moment only containing `UL17`). This has the disadvantage that you need to use the year-dependent string `UL17` in the producer code to retrieve the correct thresholds, which I think we should avoid as it makes the code less general.

Rather, I suggest removing the level, and setting the correct thresholds based on the campaign year at the time of config creation. This could go something like this:
```
cfg.x.trigger_thresholds = DotDict.wrap({
    "dijet": {
        "central": (
            [70, 87, 111, 180, 247, 310, 373, 457, 562]
            if campaign.x.year == 2017
            else [...] if campaign.x.year == 2018
            ...
            else None
        ),
        ...
    },
})
```